virttest/utils_test/libvirt.py

```diff
@@ -1164,7 +1164,7 @@ def do_migration(self, vms, srcuri, desturi, migration_type, options=None,
         :param thread_timeout: time out seconds for the migration thread running
         :param ignore_status: determine if an exception is raised for errors
         :param func: the function executed during migration thread is running
-        :param func_params: used by func
+        :param args: used by func - func_param, shell=True(bool) etc.,
```

Comment:
And it's better to describe clearly that func_params is mandatory. 
If no func_param, none is requested.

You have to consider all the circumstances for API function